<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Set_certain_time</class>
 <widget class="QDialog" name="Set_certain_time">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>533</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1021</width>
    <height>533</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1021</width>
    <height>533</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="lblDispDate_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1001</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>일정시간</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtnUseOnOff1_set_20">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>420</y>
     <width>491</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>저장</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtnUseOnOff1_set_19">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>501</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>321</width>
     <height>331</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string>시작시간</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_set_irrigation_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>321</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>70</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>_ _ : _ _</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_9">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>331</width>
     <height>331</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
   <widget class="QLabel" name="label_31">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string>종료시간</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_set_irrigation_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>331</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>70</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>_ _ : _ _</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_10">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>80</y>
     <width>331</width>
     <height>181</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string>관수횟수</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_set_irrigation_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>331</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>70</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_11">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>270</y>
     <width>331</width>
     <height>141</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
   <widget class="QLabel" name="label_33">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>7</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string>시간 간격</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_time_interval">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>75</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string>_ _ : _ _</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>65</y>
      <width>331</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_set_irrigation_3</sender>
   <signal>clicked()</signal>
   <receiver>Set_certain_time</receiver>
   <slot>set_time_start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>15</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>414</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_set_irrigation_5</sender>
   <signal>clicked()</signal>
   <receiver>Set_certain_time</receiver>
   <slot>set_time_end()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_set_irrigation_6</sender>
   <signal>clicked()</signal>
   <receiver>Set_certain_time</receiver>
   <slot>set_irrigation_time()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>708</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>721</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBtnUseOnOff1_set_19</sender>
   <signal>clicked()</signal>
   <receiver>Set_certain_time</receiver>
   <slot>set_cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>512</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>528</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBtnUseOnOff1_set_20</sender>
   <signal>clicked()</signal>
   <receiver>Set_certain_time</receiver>
   <slot>set_certain_save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>570</x>
     <y>514</y>
    </hint>
    <hint type="destinationlabel">
     <x>584</x>
     <y>528</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>set_time_start()</slot>
  <slot>set_time_end()</slot>
  <slot>set_irrigation_time()</slot>
  <slot>set_cancel()</slot>
  <slot>set_certain_save()</slot>
 </slots>
</ui>
